src/components/features/ctf-components/ctf-footer/ctf-footer.tsx

```diff
@@ -257,11 +257,13 @@ export const CtfFooter = (props: FooterFieldsFragment) => {
       <Container
         maxWidth={false}
         className={classes.footerContainer}
-        {...ContentfulLivePreview.getProps({
-          entryId: footerContent?.sys?.id,
-          fieldId: 'menuItems',
-          locale,
-        })}
+        {...(footerContent?.sys?.id
```

Comment:
this is a little hard to read. I assume we are conditional destructing props from inspectorMode, if footerContent.sys.id is defined?

Reply:
yes exactly, updated the code to be a little bit more readable 👍